Extension/package.json

```diff
@@ -561,6 +561,22 @@
                         "description": "%c_cpp.configuration.inactiveRegionBackgroundColor.description%",
                         "scope": "resource"
                     },
+                    "C_Cpp.refactor.includeHeader": {
```

Comment:
I think "refactor" should be "refactoring" to match the other VS Code references to the term "refactoring", e.g. files.refactoring.autoSave